DenoServer/auth-service/auth-handler.ts

```diff
@@ -0,0 +1,49 @@
+import { SessionUser, authService } from "./auth-service.ts";
+import { OptioPongContext } from "./auth-service.ts";
+
+export type Resource = keyof typeof resources;
+export type Action<T extends Resource> = (typeof resources)[T][number];
+
+const resources = {
+  player: ["create", "read", "update", "delete"],
+  game: ["create", "read", "update", "delete"],
+  user: ["read", "delete"],
+  roles: ["write"],
+} as const;
+
+const roles: Record<string, { resource: string; actions: "*" | string[] }[]> = {
+  admin: [
+    { resource: "player", actions: "*" },
+    { resource: "game", actions: "*" },
+    { resource: "roles", actions: "*" },
+    { resource: "user", actions: "*" },
+  ],
+};
+
+export class Auth {
+  private user: SessionUser;
+
+  constructor(context: OptioPongContext) {
```

Comment:
Kan vi rename til tennisTableContext? eller authContext kanskje for å være helt generisk